CSS/utility.css

```diff
@@ -133,14 +147,19 @@
   max-height: 9.375rem;
   overflow: scroll;
   overflow-x: hidden;
+  /* overflow-x: visible;
+  max-width: 60rem; */
```

Comment:
Suggestion: please remove commented code if not necessary.

Reply:
I have kept it for future refference